### PR TITLE
EXPANDR-5778: Organizational scope searching for EC2

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [X] In Progress
- [] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-5778

## Description
The recommended way to search for an EC2 instance by public IP is called IPAM Public IP Insights (https://aws.amazon.com/about-aws/whats-new/2023/07/aws-public-ip-insights-vpc-ip-address-manager/) and it just got API to access data (commands added in https://github.com/demisto/content/pull/31165).  This PR increases the abilities of the `AWS - Enrichment` playbook to use the AWS IPAM Public IP Insights API to search for an IP address on of an EC2 instance in the organization scope and to pull additional information on it. 

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
On hold until https://github.com/demisto/content/pull/31464 is complete

@ShirleyDenkberg , can you please review when possible?

**Comment by johnnywilkes** [comment]
@melamedbn / @adi88d , sharing demo video with you via drive

**Comment by adi88d** [comment]
@johnnywilkes the PR is ready for review?
its still a draft

**Comment by johnnywilkes** [comment]
@adi88d , still waiting on https://github.com/demisto/content/pull/31464 to merge

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you please review when possible?

File: Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml [hunk 1]
```diff
@@ -255,17 +218,447 @@ tasks:
...
+      - "16"
+      "yes":
+      - "14"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if AWS regions were returned from last command in order to proceed.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml:280]:
```suggestion
      description: Determines if AWS regions were returned from the last command in order to proceed.
```

File: Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml [hunk 2]
```diff
@@ -255,17 +218,447 @@ tasks:
...
+      - "16"
+      "yes":
+      - "17"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if AWS IPAM resources discoveries  were returned from last command in order to proceed.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml:358]:
```suggestion
      description: Determines if AWS IPAM resources discoveries were returned from the last command in order to proceed.
```

File: Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml [hunk 3]
```diff
@@ -255,17 +218,447 @@ tasks:
...
+      - "16"
+      "yes":
+      - "20"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if an AWS IPAM public IP was returned from last command in order to proceed.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml:493]:
```suggestion
      description: Determines if an AWS IPAM public IP was returned from the last command in order to proceed.
```

File: Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml [hunk 4]
```diff
@@ -255,17 +218,447 @@ tasks:
...
+      - "21"
+      "yes":
+      - "22"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determines if the AWSAssumeRoleName Input values is defined in order to proceed.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/Playbooks/AWS_-_Enrichment.yml:535]:
```suggestion
      description: Determines if the AWSAssumeRoleName Input values are defined in order to proceed.
```

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_10.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### AWS - Enrichment
+
+- Added the *AWSAssumeRoleName* input to specify role to assume for organization level user accounts.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_10.md:6]:
```suggestion
- Added the *AWSAssumeRoleName* input to specify the role to assume for organization level user accounts.
```

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_10.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+- Added support for searching for EC2 instance public IP in an organization scope using *aws-ec2-describe-ipam-resource-discoveries* and *aws-ec2-get-ipam-discovered-public-addresses* commands.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_10.md:7]:
```suggestion
- Added support for searching for the EC2 instance public IP in an organization scope using the following  commands:
   - ***aws-ec2-describe-ipam-resource-discoveries***
   - ***aws-ec2-get-ipam-discovered-public-addresses***
```

**Comment by ShirleyDenkberg** [comment]
@adi88d @melamedbn Doc review completed.

**Comment by adi88d** [comment]
Your contribution was impacted by changes in the git history of the content repo, and you see many file changes that are not related to your PR. Our team is currently working on a solution, and we hope that it will be resolved soon.

**Comment by adi88d** [comment]
Hello @johnnywilkes  ,

Last Thursday, December 20th, there was an update in the content repo. This is why your PR looks like there were a lot of commits and files changed.
To fix this issue please do the following:
1. Save all of your changes locally or in a new branch.
2. Enter your forked repository page in GitHub.
3. At the top of the page, you will probably see something like this, please ensure you are in the master branch.
image

5. Click on “Sync fork” button on the top right side of the page.
image

6. The following message will appear (This branch has conflicts that must be resolved).

7. Click on the button “Discard commits”. Note that for this action to work, you should have permission to do a force push in the forked repo. If you don’t have one, ask the forked repo Owner to do that.
8. Enter your branch and repeat the same steps (4-6).
9. Check your PR, and see if your changes have been saved. If yes - you finished the update process.
10. If your PR does not show your changes, open a new PR with the changes that you saved at the beginning of the process.
11. In case you opened a new PR please close the old one, and update your reviewer with the new PR link.
12. Please paste on the new open PR the URL to your previous PR so our team member can continue the review from the same place.

Thank you very much for your contribution and your understanding of the situation.
Have a nice holiday season!

**Comment by johnnywilkes** [comment]
@melamedbn / @adi88d , closing this PR and reopening as https://github.com/demisto/content/pull/31849

**Pull request merged**